### DISCUSSION TITLE
Unable to change PFP

### DISCUSSION THREAD

**Discussion by GDTMG232** [community/community#125953] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

**File contents don’t match the file extension.**

What does this mean in terms of profile pictures

**Comment by davevad93** [comment, marked as answer]
Hi @GDTMG232 ,

I guess it means that the picture you uploaded doesn't match the original file format, for example if the picture has a `PNG` extension and you have changed it to another extension without having converted it properly that's probably the cause of the error. Also remember that only `PNG`, `GIF` or `JPG` images are allowed as profile picture.

**Reply by GDTMG232** [reply]
It's my discord PFP and I think i need to actually convert it to PNG format from WEBP now that you mention it.

i'm gonna try that rq
